Libplanet.Tests/Common/Action/ThrowException.cs

```diff
@@ -13,14 +12,18 @@ public ThrowException()
         {
         }
 
-        public bool ThrowOnRehearsal { get; set; } = false;
+        public bool ThrowOnRehearsal { get; set; }
 
-        public bool ThrowOnExecution { get; set; } = false;
+        public bool ThrowOnExecution { get; set; }
+
+        public bool ThrowOnRendering { get; set; }
 
         public IValue PlainValue =>
-            new Bencodex.Types.Dictionary(new Dictionary<IKey, IValue>
+            new Dictionary(new Dictionary<IKey, IValue>
```

Comment:
IMHO for this one line the previous code was better because it's hard to distinguish if it's from Bencodex or .NET Framework at a first glance.

Reply:
Agree. Added a commit https://github.com/planetarium/libplanet/pull/591/commits/e225a0af78ae4176d7795aba17b1b805ddfe32cf.